src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardPlayerRequestResolver.java

```diff
@@ -91,17 +91,17 @@ public void resolve(@NotNull final IRequestManager manager, @NotNull final IRequ
                   ((BuildingBasedRequester) request.getRequester()).getBuilding().isPresent() &&
                   ((BuildingBasedRequester) request.getRequester()).getBuilding().get() instanceof AbstractBuilding)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3655)